src/Grpc/Interop/test/InteropTests/InteropTests.cs

```diff
@@ -64,7 +64,6 @@ public InteropTests(ITestOutputHelper output)
     public Task UnimplementedMethod() => InteropTestCase("unimplemented_method");
 
     [Fact]
-    [QuarantinedTest("https://github.com/dotnet/aspnetcore/issues/41248")]
```

Comment:
Policy is to leave it quarantined. Remove attribute after 30 days of passes

Reply:
Got it

Reply:
BUT this feels like one that we should un-quarantine since its my change that made it flaky.

Reply:
Maybe if it were 100% repro before, and had a targeted test to say it was 100% fixed now. Right now we only *think* it's fixed 😁, hence staying quarantined.

Reply:
It reprod 100% before on my machine.

Reply:
It's just a general rule, but up to devs judgement if they strongly believe they know the cause and the fix.

Reply:
Yea, I was able to repro locally 100%, debug and root cause the thing.